webapp/config.example.js

```diff
@@ -1,4 +1,4 @@
 var config = {
-  IRMASERVER: '',
-  EMAILSERVER: '',
+  IRMASERVER: 'http://localhost:8088',
+  EMAILSERVER: 'http://localhost:8080/irma_email_issuer-1.1.0/api',
```

Comment:
Somehow I get 404s when the webapp tries to contact the `EMAILSERVER`.

Reply:
solved: issue was caused by wrong orderoing of rules in .gitignore